code/GoogleSitemap.php

```diff
@@ -244,7 +244,7 @@ public function getItems($class, $page = 1)
             }
 
             if($redirector) {
-                foreach (ClassInfo::subclassesFor('RedirectorPage') as $redirectorClass) {
+                foreach (ClassInfo::subclassesFor('SilverStripe\\CMS\\Model\\RedirectorPage') as $redirectorClass) {
```

Comment:
The general approach in SS core is to have an import (`use` statement) and then just put `RedirectorPage::class` in place of writing out the FQCN every time